Header header

uint8[] station_num
uint8[] form
int16[] vel
uint16[] vel_ac
uint16[] vel_de
bool[] pos_mode
int32[] pos
uint16[] pos_thr